src/components/blog/components/blogAuthors.js

```diff
@@ -69,5 +70,11 @@ export default {
         name: "Igor Bobriakov",
         avatar: IgorImg,
         twitter: "WebinyCMS"
+    },
+    ben: {
+        name: "Ben Read",
+        description: "Developer Relations & Software Engineer @ Webiny",
+        avatar: BenImg,
+        twitter: "WebinyCMS"
```

Comment:
Hi @endymion1818, you can use your Twitter handle here. 
For Igor, since he wrote the guest blog and we don't had his Twitter, we used the WebinyCMS handle, thanks!

Reply:
Ah great thanks will do!